wger/core/forms.py

```diff
@@ -42,7 +42,7 @@ class Meta:
 
 class UserEmailForm(forms.ModelForm):
     email = EmailField(label=_("Email"),
-                       help_text=_("Only needed to reset your password in case you forget it."),
+                       help_text=_("Used for password resets and, optionally, workout reminders."),
```

Comment:
Actually, there are other types of emails besides workout reminders, so I'd use something more general. Perhaps something like this? *"Used for password resets and, optionally, email reminders."*